### PR TITLE
Update deleted tests

### PR DESCRIPTION
**Pull request by aditya-kumbhar**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1666,13 +1666,13 @@ https://github.com/apache/incubator-shenyu,3777209fc76800175df3962213ca1febe57ed
...
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.AssignPathManagedMergeReaderTest.testMultManagerMergeRemoteSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testAsyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testSyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,,,
+https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.MetaSimpleSnapshotTest.testSerialize,ID,Deleted,https://github.com/apache/iotdb/pull/4380,https://github.com/apache/iotdb/commit/11c6c331e61a127a3c66906d5dfde75069ec04c6
```
Comment (darko-marinov) [inline pr-data.csv:1669]: Please don't change the status for this prior PR, unless you understand whether it was `Rejected` or `Accepted` before the test was deleted. https://github.com/apache/iotdb/pull/4380 itself shows `Closed`, but at the bottom of the discussion is some `Merged`.

Comment (aditya-kumbhar) [reply pr-data.csv:1669]: I have reverted my changes and updated the Deleted tests with no prior PRs.

File: pr-data.csv [hunk 2]
```diff
@@ -1668,11 +1668,11 @@ https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d9
...
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testAsyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testSyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,,,
+https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,Deleted,,https://github.com/apache/iotdb/commit/11c6c331e61a127a3c66906d5dfde75069ec04c6
 https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.utils.SerializeUtilTest.testCreateMultiTimeSeriesPlanLog,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:1676]: Was `SerializeUtilTest` also deleted if they deleted the entire module?

Comment (aditya-kumbhar) [reply pr-data.csv:1676]: Yes it was. Updated the PR.

**Pull request merged**